lib/sync_checker/formats/publication_check.rb

```diff
@@ -57,6 +58,16 @@ def expected_details_hash(edition)
       def rendering_app
         Whitehall::RenderingApp::WHITEHALL_FRONTEND
       end
+
+      def filter_documents_for_locale(edition, locale)
```

Comment:
Yeah... I thought about that. It's a tricky one as one is essentially test code and one is part of the code under test and it seems a bit pointless to test code with itself. We've kind of done a mix of that throughout the sync checks.